### ISSUE TITLE
boolean column template error

### ISSUE THREAD

**Issue by AndreyBobrov** [Laravel-Backpack/CRUD#215]
Default boolean template uses following syntax:

`{{ trans('backpack::crud.yes', 'Yes') }}`

but Laravel trans() helper doesn't accept second argument as default value and it must be an array:

```
    /**
     * Get the translation for a given key.
     *
     * @param  string  $id
     * @param  array   $parameters
     * @param  string  $domain
     * @param  string  $locale
     * @return string|array|null
     */
    public function trans($id, array $parameters = [], $domain = 'messages', $locale = null)
    {
        return $this->get($id, $parameters, $locale);
    }
```

thus the use of boolean column causes an PHP error:

`exception 'ErrorException' with message 'Argument 2 passed to Illuminate\Translation\Translator::trans() must be of the type array, string given`

**Comment by OwenMelbz** [comment]
@tabacitu I think you added this change? Did you have a plan/know about this?

**Comment by AndreyBobrov** [comment]
fixed 5780563169e0d4bcfe7fccbbd4365f76349ba612
